pypeit/utils.py

```diff
@@ -898,7 +898,7 @@ def polyval2d(x, y, m):
         z += a * x**i * y**j
     return z
 
-
+# TODO: Any reason to keep this?
```

Comment:
I'd say this is safe to delete... the more modern version is stored in: `core.fitting.robust_fit`